draft-yasskin-http-origin-signed-responses.md

```diff
@@ -1054,112 +1054,79 @@ This allows the mitigation against downgrades in {{seccons-downgrades}}, but
 prohibits intermediates from providing a cache of the validity information. We
 could do both with a list of URLs.
 
-## application/http-exchange+cbor format for HTTP/1 compatibility # {#application-http-exchange}
+## application/signed-exchange format # {#application-signed-exchange}
+
+To allow signed exchanges to be the targets of `<link rel=prefetch>` tags, we
+define the  `application/signed-exchange` content type that represents a signed
+HTTP exchange, including request metadata and header fields, response metadata
+and header fields, and a response payload.
+
+This content type consists of the concatenation of the following items:
+
+1. The ASCII characters "sxg1" followed by a 0 byte, to serve as a file
+   signature. This is redundant with the MIME type, and receipients that receive
+   both MUST ignore the values in these first 5 bytes.
```

Comment:
I would probably suggest checking it, rather than ignoring it, just to make sure folks get it right. https://tools.ietf.org/html/draft-thomson-postel-was-wrong-02 and stuff.

Reply:
Done.